src/Microsoft.SqlTools.SqlCore/Microsoft.SqlTools.SqlCore.nuspec

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.SqlTools.SqlCore</id>
+    <version>1.0.0</version>
+    <authors>Microsoft</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="expression">MIT</license>
+    <projectUrl>https://github.com/Microsoft/sqltoolsservice/</projectUrl>
+    <description>Provides core SQL functionality for SQL server editors like Object explorer, Query Execution and Scripting</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <dependencies>
+      <dependency id="Microsoft.SqlServer.SqlManagementObjects" version="170.12.0" />
```

Comment:
This should be using the versions from the Packages.props so we don't risk forgetting to update this. You'll probably need to make a Property for these versions and then reference that in the Packages.props though.